ledger-obsolete/src/main/kotlin/net/corda/v5/ledger/obsolete/contracts/TransactionVerificationException.kt

```diff
@@ -33,26 +31,26 @@ open class TransactionResolutionException(val hash: SecureHash, message: String)
  *
  * @property hash Hash of the bytes of the attachment, see [Attachment.id]
  */
-class AttachmentResolutionException(val hash: AttachmentId) : FlowException("Attachment resolution failure for $hash")
+class AttachmentResolutionException(val hash: AttachmentId) : CordaRuntimeException("Attachment resolution failure for $hash")
 
 /**
  * A non-specific exception for the attachment identified by [attachmentId]. The context
  * for this error is provided via the [message] and [cause].
  * @property attachmentId
  */
 class BrokenAttachmentException(val attachmentId: AttachmentId, message: String?, cause: Throwable?)
-    : FlowException("Attachment $attachmentId has error (${message ?: "no message"})", cause)
+    : CordaRuntimeException("Attachment $attachmentId has error (${message ?: "no message"})", cause)
 
 /**
  * Indicates that some aspect of the transaction named by [txId] violates the platform rules. The exact type of failure
- * is expressed using a subclass. TransactionVerificationException is a [FlowException] and thus when thrown inside
- * a flow, the details of the failure will be serialised, propagated to the peer and rethrown.
+ * is expressed using a subclass. TransactionVerificationException is a [CordaRuntimeException] and thus when thrown
+ * inside a flow, the details of the failure will be serialised, propagated to the peer and rethrown.
```

Comment:
Is that behaviour still true with the change to `CordaRuntimeException`?

Reply:
Yes, that is wrong. Want me to change it, or ignore it since its in the obsolete module?